src/Error/Error.php

```diff
@@ -47,28 +47,43 @@
     private $filePath;
 
     /**
-     * @param int    $type
-     * @param string $filePath
+     * @var \Exception|\Error|\Throwable|null
      */
-    public function __construct($type, $filePath)
+    private $source;
+
+    /**
+     * @param int                               $type
+     * @param string                            $filePath
+     * @param \Exception|\Error|\Throwable|null $source
```

Comment:
PHPDoc spec or PSR doc-proposal
`@param <type> [$name] [descr]`

`<type>` is valid PHP Type. `?\Throwable` is valid PHP7.1 type.